app/views/widgets/primary_content/_funnel_1up_boxes_1.html

```diff
@@ -1,37 +1,36 @@
-            <div class="content">
-            <p>
-                <img alt="sample image 1 of 2 for funnel widget" class="featureImage" height="150" src="//www.chapman.edu/students/_files/slide-small/new-slide-region.jpg" width="200"/>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec leo turpis, dictum a quam eget, pellentesque feugiat diam. Quisque non lobortis mi. Aliquam tempor fermentum volutpat. Duis nisl urna, molestie vel arcu vitae, pulvinar hendrerit felis. Quisque at tellus egestas, suscipit metus vel, semper ex. Donec commodo sem non ultrices hendrerit. Nunc at gravida odio. Praesent elementum porta felis, eu fringilla odio placerat sit ame.</p>
-            </p>
-                            <ul class="linkList">
-                                                                                                    <li>
-                        <a href="http://www.chapman.edu/academics/learning-at-chapman/index.aspx">Internal Link »</a>
-                        </li>
-                                                                                                                        <li>
-                        <a href="http://www.chapman.edu">External Link »</a>
-                        </li>
-                                                                                                                        <li>
-                        <a href="//www.chapman.edu/campus-services/marketing-communication/_files/campus-aerial-view.jpg">File Link »</a>
-                        </li>
-                                                    </ul>
-            </div>
-                    </div>
-                                    <div class="region">
-            <h3>Funnels (1 up)</h3>  
-            <div class="content">
-            <p>
-                <img alt="sample image 2 of 2 for funnel widget" class="featureImage" height="150" src="//www.chapman.edu/campus-services/marketing-communication/_files/website/images/image-sizes/200x150.jpg" width="200"/>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec leo turpis, dictum a quam eget, pellentesque feugiat diam. Quisque non lobortis mi. Aliquam tempor fermentum volutpat. Duis nisl urna, molestie vel arcu vitae, pulvinar hendrerit felis. Quisque at tellus egestas, suscipit metus vel, semper ex. Donec commodo sem non ultrices hendrerit. Nunc at gravida odio. Praesent elementum porta felis, eu fringilla odio placerat sit amet.
-            </p>
-                            <ul class="linkList">
-                                                                                                    <li>
-                        <a href="http://blogs.chapman.edu">Blogs at Chapman »</a>
-                        </li>
-                                                    </ul>
-            </div>
-                    </div>
-            </div>
-            
+  <div class="region">
+    <h2>Diversity and Inclusion</h2>
```

Comment:
Besides the indentation, all I changed was the headings from h3 to h2 on this line and line 23.